lib/Yancy/Backend/Mysql.pm

```diff
@@ -150,6 +150,12 @@ sub dbschema {
 }
 sub filter_table { 1 }
 
+sub fixup_default {
```

Comment:
Do we need a separate `sub` here, or can we use a single "normalize" sub that would do this for us?

Reply:
My thought was to isolate things, and to name things exactly with what they do, following Kevlin Henney. If several different fixups for different things are required, my feeling is we should be explicit about each.

Reply:
Sounds good to me. If down the road a ways it turns out we're just duplicating code we can deal with it then.